ui-components/src/page_components/listing/listing_sidebar/LeasingAgent.tsx

```diff
@@ -20,6 +21,11 @@ const LeasingAgent = (props: LeasingAgentProps) => {
     ? `tel:${listing.leasingAgentPhone.replace(/[-()]/g, "")}`
     : ""
 
+  let managementWebsite = props.managementCompany?.website
+  if (managementWebsite && !managementWebsite.startsWith("http")) {
```

Comment:
@seanmalbert Thoughts about this validation being on the frontend vs the backend?

Reply:
@emilyjablonski , Ideally this is validated and handled on entry/import, but I don't think this needs to be a blocker.